app/assets/src/api/analytics.js

```diff
@@ -25,7 +25,7 @@ export const logAnalyticsEvent = async (eventName, eventData = {}) => {
         // see traits_for_segment
         admin: traits.admin,
         biohub_user: traits.biohub_user,
-        czi_user: traits.biohub_user,
```

Comment:
Does this mean you are including biohub users at czi users? So biohub users would be excluded from the list of clean users?

Reply:
Yes, after this fix, czi_user and USERS_CLEAN ought to exclude biohub_users

Reply:
Thanks for the update. I was including biohub as a segment of users_clean. I will post over in #proj-idseq-analytics

Reply:
Wait sorry, for some reason I think I misunderstood or mistyped. czi_user should definitely not overlap/include biohub_user. I think users_clean DOES include Biohub users.